app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,93 @@
+class ExpensesController < ApplicationController
+  include ChargingsControllerConcern
```

Comment:
Can I ask you to rename this concern. It is used in several controllers, not only in ChargingsController. A better name would be something like TurboStreamErrorsHandlerConcern